examples/time_freezing_hystheresis/time_optimal_control_car/car_hystheresis_model_voronoi.m

```diff
@@ -55,7 +55,7 @@
 g_3 = norm([psi;w]-z3)^2;
 g_4 = norm([psi;w]-z4)^2;
 
-g_ind = [g_1;g_2;g_3;g_4];
+model.g_ind = [g_1;g_2;g_3;g_4];
```

Comment:
It is not new, but I think the name can be better.
I first thought `ind` is for index, but it is indicator.

Reply:
I suggest `g_Stewart` or `g_switch`, as these are the switching indicator functions from Stewart's reformulation of Filippov systems.